docs/populating-configuration.md

```diff
@@ -7,23 +9,19 @@ if you need to create STANDARD dinosaur instances. Follow the guide in [Quota Ma
 ```
 make aws/setup
 ```
-4. Setup SSO configuration
-    - keycloak cert
-    ```
-    echo "" | openssl s_client -servername identity.api.stage.openshift.com -connect identity.api.stage.openshift.com:443 -prexit 2>/dev/null | sed -n -e '/BEGIN\ CERTIFICATE/,/END\ CERTIFICATE/ p' > secrets/keycloak-service.crt
-    ```
```

Comment:
This secret is not used anymore hence the removal

Reply:
:+1: